lib/features/add_property/view/add_property_screen.dart

```diff
@@ -144,6 +146,12 @@ class BottomController extends StatelessWidget {
   const BottomController(
       {required this.pageController, required this.totalPages, super.key});
 
+  String _convertSuccessTitle(String title) {
+    List<String> newTitle = title.split(' ');
+    newTitle.sort((a, b) => a.length.compareTo(b.length));
+    return '${newTitle.join(' ')}!';
+  }
+
```

Comment:
What is this for?

Reply:
@suesitran  Hi Sue, since we already have title called "Congratulation icon" in the SuccessBookingScreen but the title I need for the toast is "icon Congratulation". so the method is to swap the position of the original title.

Reply:
No, just create new string please